packages/api/src/auth/decoders/index.ts

```diff
@@ -48,17 +49,29 @@ export const decodeToken = async (
     // Make this a warning, instead of a hard error
     // Allow users to have multiple custom types if they choose to
     if (process.env.NODE_ENV === 'development') {
-      console.warn(
+      console.error(
         `The auth type "${type}" is not officially supported, we currently support: ${Object.keys(
           typesToDecoders
         ).join(', ')}`
       )
 
-      console.warn(
+      console.error(
         'Please ensure you have handlers for your custom auth in getCurrentUser in src/lib/auth.{js,ts}'
       )
     }
+
+    throw new Error(
```

Comment:
I think this locks down the implementation too much!

Reply:
Removed error. Will return now